src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -292,41 +278,24 @@ function sampleDiscussBook(imageFileName) {
   stream.write(request);
 }
 
-require(`yargs`)
+argv = require(`yargs`)
```

Comment:
@beccasaurus @alexander-fenster 
To use `yargs` in standalone samples it seems that we need to add `yargs` to `package.json` of the gapic package. Is it OK?

Reply:
No, we should not add `yargs` to the main `package.json` of gapic library. Samples must have its own `package.json` that should have this dependency.